open-metadata-implementation/user-interfaces/presentation-server/cra-server/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "cra-server",
+  "version": "1.0.0",
+  "description": "Egeria Presentation Server",
+  "main": "index.js",
+  "scripts": {
+    "start": "run-p start:server start:client",
+    "start:server": "nodemon index.js",
+    "start:client": "cd ../cra-client && npm start",
+    "build": "cd ../cra-client && npm run build",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "contributors": [
+    {
```

Comment:
This should have the egeria contributor statement in not individuals

Reply:
search for 'webpack-build' in the pom.xml. This shows that currently we run webpack with the arguments `--mode development --devtool inline-source-map` in the nodejs directory. If this no longer works, those args/executions will need updating accordingly.

the file `open-metadata-distribution/open-metadata-assemblies/src/main/assemblies/egeria-presentation-server.xml`

then controls what gets put into an archive, which is then installed into a docker container which is defined in `open-metadata-resources/open-metadata-deployment/docker/presentation-server` . It is placed in /home/node/presentation-server & we start by running `node src/server/index.js`

Reply:
this is for a 'production' build. Obviously anything additional to aid in development turnaround is fine, and at least for a UI developer, I don't see the need to support in the POM (though possible if desired)

Reply:
Updated contributor statement with standard Apache line. i am unsure about the second comment - perhaps we can discuss more after this merge? the production ready version of CRA is still not finalized yet, so i think this comment would be in line with those changes i plan on doing next